lesson26/src/js/index.js

```diff
@@ -3,6 +3,15 @@ import * as loading from "./module/loading";
 import * as slide from "./contents/slideshow";
 import * as tab from "./contents/tab";
 
+const logout = () =>  {
+    if(localStorage.getItem('token')){
+        localStorage.removeItem('token');
+        location.href = "./login.html";
+    }
+}
+
+document.querySelector('[data-id="logout-button"]').addEventListener("click", logout);
+
```

Comment:
A logout function has been added. 
If a `token` exists in local storage, the value is removed from local storage.